Tests/Parser/Client/fixtures/browser.yml

```diff
@@ -2786,8 +2786,8 @@
     type: browser
     name: Qwant Mobile
     version: "1.1"
-    engine: Gecko
-    engine_version: ""
+    engine: WebKit
```

Comment:
engine: blink
family: Chrome

Reply:
For the family part, this is the same issue as I mentioned for iOS - the family is only specified in terms of the browser name, so the family detector can't distinguish between a Gecko-based Qwant Mobile and a Chrome-based Qwant Mobile. The best option I can think of would be to set the family to null for all Qwant browsers.

The engine mismatch seems to be affecting a lot of Blink-based browsers which are all currently being detected as WebKit 537.36. I was planning to do a separate PR to handle that but it'll be a large one.

Reply:
We solved this for other browsers by creating an additional rule.
something like this:
```yml
- regex: 'Chrome/.+QwantMobile/(\d+\.[.\d]+)'
  name: 'Qwant Mobile'
  version: '$1'
  engine:
    default: 'Blink'
```

Reply:
We may need to add additional code in the future.
If the browser does not belong to the browser family,
check the presence of the words Firefox, Chrome in the user agent and compare them with the browser family
For Ios, leave browser_family: unknown
Also take care of Yandex browser_family: unknown (although it has been running on the chrome engine for long time)

this hypothesis requires detailed study.